pom.xml

```diff
@@ -59,7 +59,8 @@
 
     <commons.junit.version>5.9.1</commons.junit.version>
     <commons.japicmp.version>0.16.0</commons.japicmp.version>
-  </properties> 
+    <commons.findbugs.version>3.0.5</commons.findbugs.version>

```

Comment:
-1: Take a look at the rest of the POM and parent POM, FindBugs has been replaced by SpotBugs

Reply:
yep i'm aware findbugs got replace by spotbugs, but you still have findbugs config so i'm fixing that broken config...
i can do a pr to remove findbugs if that is your preference...